### PR TITLE
Clarify more about how SSL/TLS certificates are used

### PR DESCRIPTION
**Pull request by evanphx**

### CODE REVIEW

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 1]
```diff
@@ -28,10 +28,14 @@ Before you begin, you'll need to prepare data files and a Linux instance.
     each webhook to ignore SSL errors within your VCS provider.
   * If you do not have access to a certificate, you can use
     [Let's Encrypt](https://letsencrypt.org/getting-started/) to request one for free.
-    You will provide the path on the host system to the created certificate and key
-    when requested by the setup process.
+    This functionality is available in the settings dashboard, simply click the
+    `Automatically Request Certificate via Let's Encrypt` option.
```
Comment (KFishner) [inline content/source/docs/enterprise/private/install-installer.html.md]: noob question - is there a way we can just use Let's Encrypt automatically and not have a user ever think about providing a certificate?

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 2]
```diff
@@ -28,10 +28,14 @@ Before you begin, you'll need to prepare data files and a Linux instance.
...
 * License key (provided by HashiCorp)
 
+~> **Note:** If you use your own certificate and it is issued by a private Certificate
+   Authority, you must provide the certificate for that CA in the
+   `Certificate Authority (CA) Bundle` section of the installation. This allows services
```
Comment (KFishner) [inline content/source/docs/enterprise/private/install-installer.html.md]: do we have docs on this section anywhere?

File: content/source/docs/enterprise/private/install-installer.html.md (hunk 1 above)
Comment (evanphx) [reply content/source/docs/enterprise/private/install-installer.html.md]: We have to ask for an email to use it, but we could default to using it, yes. There are some limits on how many certs can be requested for a domain name in a period of time. I worry if it's on by default, customers accidentally hit the limit.

Comment (KFishner) [reply content/source/docs/enterprise/private/install-installer.html.md]: makes sense. I guess it's a question of what is more likely:
- A user hits the Let's Encrypt limit
- A user messes up giving TFE a certificate

The latter has shown to be very likely

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 3]
```diff
@@ -28,10 +28,15 @@ Before you begin, you'll need to prepare data files and a Linux instance.
...
+   running within PTFE to access eachother properly.
```
Comment (blalor) [inline content/source/docs/enterprise/private/install-installer.html.md]: "eachother" → "each other"

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 4]
```diff
@@ -80,6 +85,24 @@ For Linux distributions other than RHEL, check Docker compatibility:
 1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. For example, to do this with UFW run: `ufw allow in on docker0`. This rule can be added before the `docker0` interface exists, so it is best to do it now, before the Docker installation.
 1. _Optional_: Get a domain name for the instance.  If you opt to use only an IP to access the instance, enter the IP into the hostname field when prompted during the web portion of the setup.
 
+
+#### Trusting SSL/TLS Certificates
+
+The installer has a section that allows multiple certificates to be specified as trusted. These
+certificates are a `Certificate Authority (CA) Bundle` and are used to allow PTFE to connect
```
Comment (blalor) [inline content/source/docs/enterprise/private/install-installer.html.md]: "These certificates are a" → "… are contained in a"?  This parses strangely for me.

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 5]
```diff
@@ -88,9 +84,24 @@ For Linux distributions other than RHEL, check Docker compatibility:
 
 #### Trusting SSL/TLS Certificates
 
-The installer has a section that allows multiple certificates to be specified as trusted. These
-certificates are a `Certificate Authority (CA) Bundle` and are used to allow PTFE to connect
-to services that uses SSL/TLS certificates issued by private CAs.
+The installer has a section that allows multiple certificates to be specified as trusted.
+A collection certificates for trusted issuers are known as a `Certificate Authority (CA) Bundle` and are
```
Comment (blalor) [inline content/source/docs/enterprise/private/install-installer.html.md]: "A collection certificates" → "A collection _of_ certificates"

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 6]
```diff
@@ -88,9 +84,24 @@ For Linux distributions other than RHEL, check Docker compatibility:
...
+used to allow PTFE to connect to services that uses SSL/TLS certificates issued by private CAs.
```
Comment (blalor) [inline content/source/docs/enterprise/private/install-installer.html.md]: "that use"

**Pull request merged**